python/node_test.py

```diff
@@ -108,6 +110,8 @@ def create(self, endpoint, client):
 
     endpoint = node_2.node().endpoint()
 
+    assert endpoint.node_id() == await node_2.net().node_id()
```

Comment:
Quick little addition to the node test just to make sure it was exposed properly.